src/ua/nalezhytyi/hw9/Main.java

```diff
@@ -0,0 +1,35 @@
+package ua.nalezhytyi.hw9;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure circle = new Circle(10);
+        Triangle triangle = new Triangle(10, 10, 2);
+        Figure square = new Square(3);
+        returnCircleAreaAndPerimetr(circle);
+        returnSquareAreaAndPerimetr(square);
+        returnTriangleAreaAndPerimetr(triangle);
+
+    }
+    private static void returnTriangleAreaAndPerimetr(Triangle triangle){
```

Comment:
Your are displaying values, not returning
Please change method's name to more propriate